internal/printer/printer_test.go

```diff
@@ -614,7 +615,40 @@ import * as $$module3 from 'custom-element';`,
 					`const name = 'world';`},
 				styles:   []string{},
 				metadata: `{ modules: [{ module: $$module1, specifier: 'one' }, { module: $$module2, specifier: 'two' }, { module: $$module3, specifier: 'custom-element' }], hydratedComponents: [One, Two, 'my-element'], hoisted: [] }`,
-				code:     "${$$renderComponent($$result,'One',One,{\"client:load\":true,\"client:component-path\":($$metadata.getPath(One)),\"client:component-export\":($$metadata.getExport(One))},{\"default\": () => $$render`${$$renderComponent($$result,'Two',Two,{\"client:load\":true,\"client:component-path\":($$metadata.getPath(Two)),\"client:component-export\":($$metadata.getExport(Two))})}${$$renderComponent($$result,'my-element','my-element',{\"client:load\":true,\"client:component-path\":($$metadata.getPath('my-element')),\"client:component-export\":($$metadata.getExport('my-element'))})}`,})}",
+				code: `<html><head>${$$renderComponent($$result,'One',One,{"client:load":true,"client:component-path":($$metadata.getPath(One)),"client:component-export":($$metadata.getExport(One))})}
```

Comment:
This test was totally wrong! These siblings where being put into slots, should have caught this earlier.